repos/spack_repo/builtin/packages/warpx/package.py

```diff
@@ -285,8 +285,9 @@ def _get_input_options(self, dim, post_install):
         # test openPMD output if compiled in
         if "+openpmd" in spec:
             cli_args.append("diag1.format=openpmd")
-            # RZ: thetaMode output uses different variables
-            cli_args.append("diag1.fields_to_plot=Er Et Ez Br Bt Bz jr jt jz rho")
+            if dim == "rz":
```

Comment:
Setting `fields_to_plot` should be removed from here since if it is unset, it is given a default value that already includes the correct fields (except rho).